src/amber/cli/commands/routes.cr

```diff
@@ -104,8 +112,8 @@ module Amber::CLI
 
       private def set_pipe(pipe_string)
         if route_match = pipe_string.to_s.match(PIPE_SCOPE_REGEX)
-          @current_pipe = route_match[1]?
-          @current_scope = route_match[2]?
+          @current_pipe = route_match[2]?
+          @current_scope = route_match[3]?
```

Comment:
Now I'm guessing those Regexs did change. Is there an extra capture group now?

Reply:
Changed due to your #1 question above (my answer #2 there), so that's why the number increased.